#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile.patch.dpatch by Ryan Finnie <ryan@finnie.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Clean up tarball Makefile

@DPATCH@

diff -ruN grepcidr-1.3-orig/Makefile grepcidr-1.3/Makefile
--- grepcidr-1.3-orig/Makefile	2005-04-19 20:19:14.000000000 -0700
+++ grepcidr-1.3/Makefile	2006-10-05 01:30:47.000000000 -0700
@@ -3,7 +3,8 @@
 #
 
 # Set to where you'd like grepcidr installed
-INSTALLDIR=/usr/local/bin
+PREFIX=/usr/local
+BIN=$(PREFIX)/bin
 
 # Set to your favorite C compiler and flags
 CC=gcc
@@ -11,13 +12,14 @@
 
 # End of settable values
 
-grepcidr:
-	$(CC) $(CFLAGS) -o grepcidr grepcidr.c getopt.c
+grepcidr: grepcidr.c
+	$(CC) $(CFLAGS) -o grepcidr grepcidr.c
 
 all:	grepcidr
 
 install:	grepcidr
-	cp grepcidr $(INSTALLDIR)
+	install -d $(DESTDIR)$(BIN)
+	install ./grepcidr $(DESTDIR)$(BIN)
 
 clean:
 	rm -f grepcidr *.o
